sdks/python/apache_beam/ml/inference/pytorch_inference.py

```diff
@@ -234,16 +263,17 @@ def run_inference(
     # If elements in `batch` are provided as a dictionaries from key to Tensors,
     # then iterate through the batch list, and group Tensors to the same key
     key_to_tensor_list = defaultdict(list)
-    for example in batch:
-      for key, tensor in example.items():
-        key_to_tensor_list[key].append(tensor)
-    key_to_batched_tensors = {}
-    for key in key_to_tensor_list:
-      batched_tensors = torch.stack(key_to_tensor_list[key])
-      batched_tensors = _convert_to_device(batched_tensors, self._device)
-      key_to_batched_tensors[key] = batched_tensors
-    predictions = model(**key_to_batched_tensors, **inference_args)
-    return [PredictionResult(x, y) for x, y in zip(batch, predictions)]
+    with torch.no_grad():
```

Comment:
Necessary to prevent `RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0!` warning

Reply:
Let's capture this information in a way that makes it easier to find for someone reading the code.

Ideally, there would be a GH issue mentioning the error `Setting to CPU due to an exception...`, and you add a comment and reference the issue, and link this PR to the issue.

Another option is to capture this info in a pull request description for this PR (slightly more difficult to find, but easier than trying to find  this review comment chain).